### PR TITLE
get-npm-package-version 1.0.0

### PR DESCRIPTION
**Pull request by lafiosca**
TagCheck

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [✅] __I will not move an already shared step version's tag to another commit__
- [✅] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [✅] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [✅] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [✅] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by lafiosca** [comment]
I would like to mention my primary use case for this step: in a React Native project, I want to use the version found in the root `package.json` as an input to other steps which set the Android versionName and iOS bundle version. Currently I'm accomplishing this with a generic script step which uses envman, but I thought it might be nice to have a custom step.

File: steps/get-npm-package-version/1.0.0/step.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+title: Get npm package version
+summary: |
+  Sets an environment variable with the npm package version
+description: |
+  Sets environment variable `$PACKAGE_VERSION` with the value found in the "version" field of package.json, for use in later steps
```
Comment (trapacska) [inline steps/get-npm-package-version/1.0.0/step.yml]: Typo in `$PACKAGE_VERSION` => `$NPM_PACKAGE_VERSION`

**Comment by trapacska** [comment]
Please fix the only typo and do a fresh release.

Thank you for the PR! 🎉

File: steps/get-npm-package-version/1.0.0/step.yml (hunk 1 above)
Comment (lafiosca) [reply steps/get-npm-package-version/1.0.0/step.yml]: Whoops, good catch! I originally had it as `$PACKAGE_VERSION` everywhere then updated it to `$NPM_PACKAGE_VERSION` to be more explicit, but I missed the description and README.

**Pull request closed without merging**